### ISSUE TITLE
Clean up headers (buttons / titles)

### ISSUE THREAD

**Issue by zekone** [NYCNinjas/awesomeapp#25]
Currently there are too many buttons in the header navigation on most pages and it doesn't allow the header to render normally. 

- [ ] Use only icons (no text) for header buttons
- [ ] Remove logout button from all headings and just have it on the Profile page as a simple button at the bottom of the page. 

**Song List page**
- [ ] Upload icon on left side (possible icons: cloud with up arrow, microphone)
- [ ] Profile icon on right side (possible icons: user silhouette, circle or not)

**Comment by HSatwick** [comment]
what about titles?

**Comment by zekone** [comment]
No, I think what we want to do is use icons that are so accurate, descriptive, and familiar that the user should be able to navigate. The profile icon should be easy to understand, and the New Upload icon should be the only other clickable action.
